pkgs/development/tools/analysis/radare2/update.py

```diff
@@ -18,50 +20,107 @@ def sh(*args: str) -> str:
 
 
 def prefetch_github(owner: str, repo: str, ref: str) -> str:
-    return sh("nix-prefetch-url", "--unpack",
-              f"https://github.com/{owner}/{repo}/archive/{ref}.tar.gz")
+    return sh(
+        "nix-prefetch-url",
+        "--unpack",
+        f"https://github.com/{owner}/{repo}/archive/{ref}.tar.gz",
+    )
 
 
-def main() -> None:
+def get_radare2_rev() -> str:
     url = "https://api.github.com/repos/radare/radare2/releases/latest"
     with urllib.request.urlopen(url) as response:
         release = json.load(response)  # type: ignore
-    version = release["tag_name"]
-    with tempfile.TemporaryDirectory() as dirname:
+    return release["tag_name"]
+
+
+def get_r2_cutter_rev() -> str:
+    url = "https://api.github.com/repos/radareorg/cutter/contents/"
```

Comment:
This should be the current version of cutter no the latest. I probably need to fix this in future.